client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -1201,4 +1205,31 @@ public void testGetIndexTemplate() throws Exception {
             new GetIndexTemplatesRequest().names("the-template-*"), client.indices()::getTemplate, client.indices()::getTemplateAsync));
         assertThat(notFound.status(), equalTo(RestStatus.NOT_FOUND));
     }
+
+    public void testPutStoredScript() throws Exception {
```

Comment:
I think the same applicable for tests for this REST API call - `RestHighLevelClientTests` is better place for it as this call doesn't belong to any category

Reply:
Maybe it is a good idea to move `putStoredScript` to other place, but I am not really sure whether `RestHighLevelClient` is a good place or not. 

But according to #27205 it seems that script API should be categorized as Indices API. I am confused...

Reply:
good point... 

in fact in https://github.com/elastic/elasticsearch/issues/27205 it is under _indices api_ while [rest api spec](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/put_script.json) says it's out of any category... 

in my PR (gonna raise it soon) I put them into `StoredScriptsIT` - to group them reasonably...

@javanna do you have any objections / concerns / suggestion ?

Reply:
RestHighLevelClientTests is a unit test while this is an integ test, adding them to a new StoredScriptsIT sounds good to me. I will move the API to the right category in the meta issue, it should not be under indices, the transport client even exposed it up until now under cluster, but we should follow our rest spec so we are aligned with the other language clients.

Reply:
and I assume the same is applicable for `StoredScriptsDocumentationIT`

Reply:
yes that sounds good too